std_msgs/Header header
int16 frameClass

int16 mode
geometry_msgs/Vector3 vel_rate
geometry_msgs/Vector3 vel_rate_fix
geometry_msgs/Vector3 vel_rate_integ
float32 quality

uint8 dir
uint16 angle
float32 dis

geometry_msgs/Vector3 acc_rate
geometry_msgs/Vector3 gry_rate

geometry_msgs/Quaternion quat


